src/sentry/rules/processing/delayed_processing.py

```diff
@@ -17,9 +21,6 @@
 logger = logging.getLogger("sentry.rules.delayed_processing")
 
 
-PROJECT_ID_BUFFER_LIST_KEY = "project_id_buffer_list"
```

Comment:
I just moved this for circular import reasons, it's kinda easier if it's in the processor